src/common/utils/validator.js

```diff
@@ -224,29 +225,51 @@ function currentOrFutureDate(dObj) {
  * * @returns {Bool} Date is within acceptable range
  */
 function dateNotTooFarInFuture(dObj) {
-  if (dObj === null || dObj === undefined) return false;
-
   const now = new Date();
-  var nextMonth = new Date(now.getFullYear(), now.getMonth() + 1, now.getDate());
+  const nextMonth = new Date(now.getFullYear(), now.getMonth() + 1, now.getDate());
+  const providedDate = getDateFromDynamicInput(dObj);
+
+  return providedDate && providedDate <= nextMonth;
```

Comment:
This might return null instead of a boolean if the providedDate is null. Please can you check

Reply:
wrappend in Boolean()